src/overlays/actors/ovl_En_Po_Fusen/z_en_po_fusen.h

```diff
@@ -15,17 +17,17 @@ typedef struct EnPoFusen {
     /* 0x144 */ SkelAnime anime;
     /* 0x188 */ EnPoFusenActionFunc actionFunc;
     /* 0x18C */ ColliderSphere collider;
-    /* 0x1E4 */ Vec3s jointTable[0xA];
-    /* 0x220 */ Vec3s morphTable[0xA];
+    /* 0x1E4 */ Vec3s jointTable[POE_BALLOON_LIMB_MAX];
+    /* 0x220 */ Vec3s morphTable[POE_BALLOON_LIMB_MAX];
     /* 0x25C */ s16 randScaleChange;
     /* 0x25E */ s16 randXZRotChange;
     /* 0x260 */ s16 randYRotChange;
     /* 0x262 */ s16 avgBaseRotation;
-    /* 0x264 */ s16 limb3Rot;
+    /* 0x264 */ s16 limbRotRightUpperArmY;
```

Comment:
This is the Y-rotation of the right upper arm, correct? I think this name could be improved a bit, since the `Y` part of the name and the `limbRot` part are very separated currently. Maybe something like `limbRotYRightUpperArm` or `rightUpperArmLimbRotY`? Could maybe even drop the `Limb` from that last one and do `rightUpperArmRotY`. Just tossing some ideas out there, but I'll leave the final choice up to you; just make sure the other `limbRot` stuff looks similar.